rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/get/120_stored_fields_ignored.yml

```diff
@@ -0,0 +1,162 @@
+---
+"_ignored field through get api using stored_fields":
+  - do:
+      indices.create:
+        index: test
+        body:
+          mappings:
+            properties:
+              keyword:
+                type: keyword
+                ignore_above: 5
+              ip:
+                type: ip
+                ignore_malformed: true
+              value:
+                type: long
+                ignore_malformed: true
+
+  - do:
+      index:
+        index:   test
+        id:      1
+        refresh: true
+        body:
+          keyword: foo
+          ip: 192.168.0.1
+          value: 23
+  - do:
+      index:
+        index:   test
+        id:      2
+        refresh: true
+        body:
+          keyword: foobar
+          ip: garbage
+          value: missing
+  - do:
+      index:
+        index:   test
+        id:      3
+        refresh: true
+        body:
+          keyword:
+            - foo
+            - bar
+            - foobar
+          ip:
+            - 10.10.1.1
+            - 192.8.1.2
+            - 199.199.300.999
+          value:
+            - 1
+            - 2
+            - ops
+
+  - do:
+      get:
+        index: test
+        id:    1
+
+  - match: {_index: "test"}
+  - match: {_id: "1"}
+  - match: {_version: 1}
+  - match: {found: true}
+  - match:
+      _source:
+        keyword: foo
+        ip: 192.168.0.1
+        value: 23
+
+  - is_false: fields
+  - is_false: stored_fields
+
+  - do:
+      get:
+        index: test
+        id: 2
+  - match: { _index: "test" }
+  - match: { _id: "2" }
+  - match: { _version: 1 }
+  - match: { found: true }
+  - match:
+      _source:
+        ip: garbage
+        keyword: foobar
+        value: missing
+
+  - is_false: fields
+  - is_false: stored_fields
+
+  - do:
+      get:
+        index: test
+        id: 3
+  - match: { _index: "test" }
+  - match: { _id: "3" }
+  - match: { _version: 1 }
+  - match: { found: true }
+  - match:
+      _source:
+        ip:
+          - 10.10.1.1
+          - 192.8.1.2
+          - 199.199.300.999
+        keyword:
+          - foo
+          - bar
+          - foobar
+        value:
+          - 1
+          - 2
+          - ops
+
+  - is_false: fields
+  - is_false: stored_fields
```

Comment:
I wonder about this specific assertion: is `stored_fields` a section of the get API response?